From 68fa7451515054e270a7c71941395ead799b219f Mon Sep 17 00:00:00 2001
From: Sanghee Kim <sh0130.kim@samsung.com>
Date: Mon, 6 Jul 2020 14:28:26 +0900
Subject: [PATCH] mbedtls: esp_config: added
 CONFIG_STDK_IOT_CORE_MBEDTLS_USE_ALT

To use iot_security_pk_xxx as eckey alternative functions
with following configuration.
- MBEDTLS_ECKEY_IOT_ALT

Signed-off-by: Sanghee Kim <sh0130.kim@samsung.com>
---
 components/mbedtls/port/include/mbedtls/esp_config.h | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/components/mbedtls/port/include/mbedtls/esp_config.h b/components/mbedtls/port/include/mbedtls/esp_config.h
index 89cdef892..a607dd570 100644
--- a/components/mbedtls/port/include/mbedtls/esp_config.h
+++ b/components/mbedtls/port/include/mbedtls/esp_config.h
@@ -2714,6 +2714,11 @@
 #include MBEDTLS_USER_CONFIG_FILE
 #endif
 
+
+#ifdef CONFIG_STDK_IOT_CORE_MBEDTLS_USE_ALT
+#define MBEDTLS_ECKEY_IOT_ALT
+#endif
+
 #include "mbedtls/check_config.h"
 
 #endif /* MBEDTLS_CONFIG_H */
-- 
2.17.1

